web3.md

```diff
@@ -71,15 +74,56 @@ module WEB3
     rule <k> #loadRPCCall(#EOF) => #shutdownWrite(SOCK) ~> #close(SOCK) ~> accept() ... </k>
          <web3clientsocket> SOCK </web3clientsocket>
 
+    rule <k> #loadRPCCall([ _, _ ] #as J) => #loadFromBatch ... </k>
+         <batch> _ => J </batch>
+         <web3response> _ => .List </web3response>
+
+    rule <k> #loadRPCCall(_:String #Or null #Or _:Int #Or [ .JSONList ]) => #sendResponse("error": {"code": -32600, "message": "Invalid Request"}) ... </k>
+         <callid> _ => null </callid>
+
+    syntax KItem ::= "#loadFromBatch"
+ // ---------------------------------
+    rule <k> #loadFromBatch ~> _ => #loadRPCCall(J) </k>
+         <batch> [ J , JS => JS ] </batch>
+
+    rule <k> #loadFromBatch ~> _ => #putResponse(List2JSON(RESPONSE), SOCK) ~> getRequest() </k>
+         <batch> [ .JSONList ] </batch>
+         <web3clientsocket> SOCK </web3clientsocket>
+         <web3response> RESPONSE </web3response>
+      requires size(RESPONSE) >Int 0
+
+    rule <k> #loadFromBatch ~> _ => getRequest() </k>
+         <batch> [ .JSONList ] </batch>
+         <web3response> .List </web3response>
+
+    syntax JSON ::= List2JSON(List)           [function]
+                  | List2JSON(List, JSONList) [function, klabel(List2JSONAux)]
+ // --------------------------------------------------------------------------
+    rule List2JSON(L) => List2JSON(L, .JSONList)
+
+    rule List2JSON(L ListItem(J), JS) => List2JSON(L, J, JS)
```

Comment:
This is confusing on first read, can we do `List2JSON(L, (J, JS))` on the RHS instead?